### PR TITLE
Add table pagerduty_incident_log. Closes #5

### LINKED ISSUE #5: Add table pagerduty_incident_log
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

### PR DESCRIPTION
**Pull request by Subhajit97**
# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: pagerduty/table_pagerduty_incident_log.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+//// TABLE DEFINITION
+
+func tablePagerDutyIncidentLog(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "pagerduty_incident_log",
+		Description: "Records log entries for the specified incident.",
+		List: &plugin.ListConfig{
+			Hydrate: listPagerDutyIncidentLogs,
+			KeyColumns: []*plugin.KeyColumn{
```
Comment (cbruno10) [inline pagerduty/table_pagerduty_incident_log.go:22]: Are there other optional quals we can add here? Looking at https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE0Nw-list-log-entries-for-an-incident, does it make sense to have any around the `include` or `is_overview` query params?

File: docs/tables/pagerduty_incident_log.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Table: pagerduty_incident_log
+
+Retrieves the record of all log entries for the specified incident.
+
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md:8]: We should add a line here about the required key qual (incident ID), similar to tables in GitHub and other plugins.

File: docs/tables/pagerduty_incident_log.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Table: pagerduty_incident_log
+
+Retrieves the record of all log entries for the specified incident.
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md]:
```suggestion
Retrieves the log entries for the specified incident.
```

File: docs/tables/pagerduty_incident_log.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Examples
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md:9]: Can you please add an example where we list a subset of incidents from the `pagerduty_incident` table, and then join those incident IDs with this table to get the logs for each incident?

File: docs/tables/pagerduty_incident_log.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  created_at,
+  agent ->> 'summary' as created_by
+from
+  pagerduty_incident_log
+where
+  incident_id = 'Q0FH5K82AJ101C'
+  and created_at > now() - interval '3 days';
+```
+
+### List all activities performed by users
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md]:
```suggestion
### List incident log entries for activities performed by users
```

File: docs/tables/pagerduty_incident_log.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  agent ->> 'summary' as user_name
+from
+  pagerduty_incident_log
+where
+  incident_id = 'Q0FH5K82AJ101C'
+  and created_at > now() - interval '3 days'
+  and agent ->> 'type' = 'user_reference';
+```
+
+### List all activities performed by service
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md]:
```suggestion
### List incident log entries for activities performed by services
```

File: docs/tables/pagerduty_incident_log.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  id,
+  incident_id,
+  created_at,
+  jsonb_pretty(channel) as action,
+  agent ->> 'summary' as user_name
+from
+  pagerduty_incident_log
+where
+  incident_id = 'Q0FH5K82AJ101C'
+  and created_at > now() - interval '3 days'
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md]: Is there a reason we limit the incident logs we retrieve to 3 days? How many incident logs/how far back do they usually go for a given incident?

File: docs/tables/pagerduty_incident_log.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
+# Table: pagerduty_incident_log
+
+Retrieves the record of all log entries for the specified incident.
+
+## Examples
+
+### List all activities on specified incident in last 3 days
```
Comment (cbruno10) [inline docs/tables/pagerduty_incident_log.md]:
```suggestion
### List incident logs for an incident from the last 3 days
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comment, thanks!

File: docs/tables/pagerduty_incident_log.md (hunk 7 above)
Comment (Subhajit97) [reply docs/tables/pagerduty_incident_log.md]: @cbruno10 there is no specific reason to limit the date range to 3 days. API doesn't have any specific max. limit set on the date range.

Comment (cbruno10) [reply docs/tables/pagerduty_incident_log.md]: @Subhajit97 Can you please remove the `created_at` clauses in this example, and the one below?

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**